layers/+filetree/treemacs/README.org

```diff
@@ -53,7 +54,7 @@ layer variable =treemacs-use-follow-mode= to non-nil.
     (treemacs :variables treemacs-use-follow-mode t)))
 #+END_SRC
 
-Default value is =t=.
+Default is =t=.
```

Comment:
To be honest, I don't really care about this particular change. But I see both styles are being used in Spacemacs, so I don't even know what to do about it. 😸

Reply:
Less words, same meaning. And as you said, both styles are already present.